cli/replicaset/cartridge.go

```diff
@@ -25,6 +25,10 @@ type cartridgeTopology struct {
 	Provider string
 	// Replicasets is an array of replicasets.
 	Replicasets []Replicaset
+	// IsCritical indicates whether instance has critical issues.
+	IsCritical bool `mapstructure:"is_critical"`
+	// isBootstrapped indicates whether instance is bootstrapped.
+	isBootstrapped bool
```

Comment:
The structure is `private` at all.
```suggestion
	// IsBootstrapped indicates whether instance is bootstrapped.
	IsBootstrapped bool
```

Reply:
Thank you! Changed.